src/main/java/org/jabref/logic/cleanup/URLCleanup.java

```diff
@@ -35,11 +39,34 @@ public List<FieldChange> cleanup(BibEntry entry) {
                 + "<>\\\\]+\\)))*\\))+(?:\\(([^\\s()<>\\\\]+|(\\([^\\s()<>\\\\]+\\"
                 + ")))*\\)|[^\\s`!()\\[\\]{};:'\".,<>?«»“”‘’]))";
 
-        final Pattern pattern = Pattern.compile(urlRegex, Pattern.CASE_INSENSITIVE);
-        final Matcher matcher = pattern.matcher(noteFieldValue);
+        String dateTermsRegex = "Accessed on|Visited on|Retrieved on|Viewed on";
+
+        /*
+         * dateRegex matches several date formats. Explanation:
```

Comment:
Please reference JabRefs Date class that there are more patterns available. And why it was not possible to extract the RegEx there into a `public static` constant.

(Deep link: https://github.com/JabRef/jabref/blob/39c31afabce00ee093c086b7586e3f4b31ac30ae/src/main/java/org/jabref/model/entry/Date.java#L137)